Core/src/OffConverter/OffReader.cpp

```diff
@@ -21,20 +21,17 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 #include <fstream>
 #include <iostream>
 
-using OpenInfraPlatform::Core::OffConverter::OffReader;
-using OpenInfraPlatform::Core::OffConverter::OffModel;
-
-OffReader::OffReader::OffReader()
+OpenInfraPlatform::Core::OffConverter::OffReader::OffReader()
```

Comment:
I thought that you've produced a namespace.h with the defines for namespace?

Reply:
Resolved: bb652dc